projects/docs/docs/commands/generate.md

```diff
@@ -14,18 +14,30 @@ and Xcode projects and workspaces as a implementation detail to edit your projec
 
 To generate the project in the current directory, youn can simply run:
 
-```
+```bash
 tuist generate
 ```
 
-The command accepts the `--path` argument, which can be used to generate the project in a different directory.
+Moreover, if external dependencies exist in the [cache](/building-at-scale/caching/), Tuist replaces them with their pre-compiled version.
+
+In large Xcode projects that contain many targets and schemes, Xcode can be slow indexing the project.
+Moreover, the build system, which needs to resolve implicit dependencies, might take longer to do so because there are more Xcode objects to analyze.
```

Comment:
```suggestion
The build system, which needs to resolve implicit dependencies, might take longer to do so because there are more Xcode objects to analyze.
```

Let's get rid of the excessive usage of `moreover` which does not convey a lot of meaning 🙂